packages/cli/src/utils/transformers/transform-css-vars.ts

```diff
@@ -146,38 +146,42 @@ export function applyColorMapping(
 
   // Build color mappings.
   const classNames = input.split(" ")
-  const lightMode: string[] = []
-  const darkMode: string[] = []
+  const lightMode = new Set<string>()
+  const darkMode = new Set<string>()
   for (let className of classNames) {
     const [variant, value, modifier] = splitClassName(className)
     const prefix = PREFIXES.find((prefix) => value?.startsWith(prefix))
     if (!prefix) {
-      if (!lightMode.includes(className)) {
-        lightMode.push(className)
+      if (!lightMode.has(className)) {
+        lightMode.add(className)
       }
       continue
     }
 
     const needle = value?.replace(prefix, "")
     if (needle && needle in mapping.light) {
-      lightMode.push(
+      lightMode.add(
         [variant, `${prefix}${mapping.light[needle]}`]
           .filter(Boolean)
           .join(":") + (modifier ? `/${modifier}` : "")
       )
 
-      darkMode.push(
+      darkMode.add(
         ["dark", variant, `${prefix}${mapping.dark[needle]}`]
           .filter(Boolean)
           .join(":") + (modifier ? `/${modifier}` : "")
       )
       continue
     }
 
-    if (!lightMode.includes(className)) {
-      lightMode.push(className)
+    if (!lightMode.has(className)) {
+      lightMode.add(className)
     }
   }
 
-  return lightMode.join(" ") + " " + darkMode.join(" ").trim()
+  return (
+    Array.from(lightMode).join(" ") +
```

Comment:
🔧 Can be simplified to `return [...lightMode, ...darkMode].join(' ').trim()`.

Reply:
I like the solution you shipped with the latest commit though. Much cleaner now!